lib/sidekiq/worker_killer.rb

```diff
@@ -12,17 +12,20 @@ class WorkerKiller
 
     def initialize(options = {})
       @max_rss         = (options[:max_rss]         || 0)
-      @grace_time      = (options[:grace_time]      || 15 * 60)
+      @grace_time      = (options.key?(:grace_time) ? options[:grace_time] : 15 * 60)
```

Comment:
I think it would be better to have a consistent way of initializing variables here. This said I'd lean more in favor of the `#key?` option since it avoids unexpected behavior such as:

```ruby
expect_false_but_obviously_is_true = false || true
# This would impact the `gc` option for instance
```

What do you think?

Reply:
I think you are right. It is my bad. Best option is `options.fetch(:grace_time, 15 * 60)`, and it is supported since Ruby 2.0.0. I'll change to consisted way here.